diff --git a/app/src/main/java/com/topjohnwu/magisk/core/App.kt b/app/src/main/java/com/topjohnwu/magisk/core/App.kt
index 2acf21c..d4842c0 100644
--- a/app/src/main/java/com/topjohnwu/magisk/core/App.kt
+++ b/app/src/main/java/com/topjohnwu/magisk/core/App.kt
@@ -67,7 +67,9 @@ open class App() : Application() {
     }
 
     override fun onConfigurationChanged(newConfig: Configuration) {
-        resources.updateConfig(newConfig)
+        if (resources.configuration.diff(newConfig) != 0) {
+            resources.updateConfig(newConfig)
+        }
         if (!isRunningAsStub)
             super.onConfigurationChanged(newConfig)
     }
